include/deal.II/grid/grid_generator.h

```diff
@@ -747,7 +747,10 @@ namespace GridGenerator
    * after one refinement is optimized.
    *
    * This function is declared to exist for triangulations of all space
-   * dimensions, but throws an error if called in 1d.
+   * dimensions, but throws an error if called if `dim==1`. If `spacedim>dim`
+   * (which is only possible if `dim==2, spacedim==3`) then the function
+   * creates a circular disk for which all vertices have a $z$ value equal
+   * to the $z$ coordinate of the center point provided by the second argument.
```

Comment:
The implementation shows that `z`-coordinate is always 0, though.

Reply:
No, not true. We compute the vertex locations as `p + something` where `p` is the center. The something has zero z component.